srcpkgs/gamemode/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'gamemode'
+pkgname=gamemode
+version=1.5.1
+revision=1
+build_style=meson
+configure_args="-Dwith-systemd=false -Dwith-dbus-service-dir=/usr/share/dbus-1/services --prefix=/usr --buildtype debugoptimized"
```

Comment:
`--prefix=/usr` is set by `build_style=meson`, not sure about build type, its also set by the build style but set to `plain`.